x/oracle/types/params_test.go

```diff
@@ -8,6 +8,69 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+func TestParamKeyTable(t *testing.T) {
+	require.NotNil(t, ParamKeyTable())
+}
+
+func TestValidateVotePeriod(t *testing.T) {
+	require.ErrorContains(t, validateVotePeriod("invalidUint64"), "invalid parameter type: string")
+	require.ErrorContains(t, validateVotePeriod(uint64(0)), "vote period must be positive: 0")
+	require.Nil(t, validateVotePeriod(uint64(10)))
```

Comment:
These should be wrapped at 80 and could be a bit more readable: 
```suggestion
	err := validateVotePeriod("invalidUint64")
	require.ErrorContains(t, err, "invalid parameter type: string")
	err = validateVotePeriod(uint64(0))
	require.ErrorContains(t, err, "vote period must be positive: 0")
	err = validateVotePeriod(uint64(10))
	require.Nil(t, err)
```